app/src/main/java/dev/arkbuilders/arkmemo/preferences/MemoPreferencesImpl.kt

```diff
@@ -23,18 +22,7 @@ class MemoPreferencesImpl @Inject constructor(@ApplicationContext context: Conte
         }
     }
 
-    override fun getPathString() = sharedPreferences.getString(CURRENT_NOTES_PATH, null)
+    override fun getPath() = sharedPreferences.getString(CURRENT_NOTES_PATH, null)
 
-    override fun getPath(): Path? {
-        val pathString = getPathString()
-        var path: Path? = null
-        try {
-            val file = File(pathString!!)
-            file.mkdir()
-            path = file.toPath()
-        } catch (e: Exception) {
-            e.printStackTrace()
-        }
-        return path
-    }
+    override fun getNotesStorage() = Path(getPath()!!)
```

Comment:
Instead of a non-null assertion here which could cause a runtime crash in theory, what about updating the `getPath`'s default return value to `empty` string instead of `null`?

Reply:
Sounds good